### ISSUE TITLE
LDAP Bind address lost (reset to 127.0.0.1) after upgrade from 6.1 to 6.2

### ISSUE THREAD

**Issue by mpantel** [zammad/zammad-docker-compose#391]
# Infos

* Docker version: 20.10.13, build a224086
* Docker-compose version: 1.27.4, build 40524192
* Operating system (Docker host): Ubuntu 22.04.3 LTS

# Expected behavior

A after upgrading Zammd 6.1 to 6.2 using docker-compose (docker-compose stop && git pull && docker-compose pull && docker-compose up) the settings in LDAP integration module should persist.

# Actual behavior

* Upgrade progresses as expected but
* LDAP failed to bind to 127.0.0.1:389 and some users could not logging
* After reinspecting the LDAP module setup and resetting the ldap server to the correct address, the system was operational again, and all other settings were intact (even ldap user and password).

# Steps to reproduce the behavior

* Install Zammad version 6.1 through docker
* setup the LDAP integration module to some existing LDAP server other than 127.0.0.1
* Follow the process to upgrade to version 6.2 as in the documentation.

**Comment by mgruner** [comment]
@fliebe92 this does not sound like a docker related issue. We did have a migration dealing with LDAP source configuration in 6.2, could this be related somehow?

**Comment by mpantel** [comment]
Hi, @mgruner 
I cannot verify whether this is a docker issue. The fact is that I upgraded the installation following the docker workflow. I didn't mess with any configuration file by hand, except some changes at the end of .env:
```
RAILS_TRUSTED_PROXIES=['127.0.0.1', '::1','0.0.0.0']
NGINX_SERVER_SCHEME=https
```
Thanks!

**Comment by mgruner** [comment]
@mpantel yes, thanks. I tend to think so because this is handled in the DB on application level. Perhaps we should move this issue to https://github.com/zammad/zammad, but let's see what @fliebe92 and @dominikklein think about it.

**Comment by fliebe92** [comment]
@mgruner Based on the given information in this report, I cannot say if the migration is faulty or not.

`preferences.delete('host_url')` could point to an issue, but without knowing the details about the LDAP source... no chance. We need to know the state before the migration.

**Comment by dominikklein** [comment]
With the current information, we can not say something.

We would need the LDAP config (sensitive stuff should be removed/replaced, important is `host`/`host_url`/`port`) before the migration and afterward:

`LdapSource.all`
